templates/pgbouncer/pgbouncer-networkpolicy.yaml

```diff
@@ -36,6 +36,10 @@ spec:
           tier: monitoring
           component: prometheus
           release: {{ .Values.platform.release }}
+    - namespaceSelector: {}
+      podSelector:
+        matchLabels:
+          app: keda-operator
```

Comment:
@dimberman can we use a similar selector scheme as our other stuff?

```
tier: keda
component: operator
release: {{ .Values.platform.release }}
```

Reply:
@schnie I'm not seeing anything in the KEDA helm chart that allows for custom labels :(. I'm gonna try to PR that as it seems like an oversight.

Reply:
https://github.com/kedacore/charts/pull/24/files I've added a PR to the KEDA charts.